README.adoc

```diff
@@ -116,7 +116,7 @@ public interface TestRetryTaskExtension {
      * <p>
      * This setting has no effect if {@link Test#getIgnoreFailures()} is set to true.
      *
-     * @return whether tests that initially fails and then pass on retry should fail the task
+     * @return whether tests that initially fail and then pass on retry should fail the task
```

Comment:
🤔 Can you please also fix the comment in the `TestRetryTaskExtension.java` class?